CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 6.0.2
+
+- Overwrite the cache event path
```

Comment:
would be nice to describe what this means for users

Reply:
Sure, I tried to be a bit more descriptive